### ISSUE TITLE
csvstack is column agnostic and corrupts output

### ISSUE THREAD

**Issue by ulfh** [wireservice/csvkit#1233]
csvstack does not check or use column names when stacking csv files. This means that reoredered columns, missing columns, or other similar changes will corrupt the produced csv output with no warning. In essence, everything that makes a tool valuable to use is nullified by this issue.

For instance, merging these two files happily produces the result below. Clearly at least a warning should be issued.

```
a,b
1,2
```
```
b,a
1,2
```
```
a,b
1,2
1,2
```

csvstack also (unsurprisingly) does not check the number of columns either. This results in corrupted csv files if the number of columns varies between the input data.

```
$ csvstack --version
csvstack 1.0.6
```

**Comment by jpmckinney** [comment]
See #245. We can issue a warning.

**Comment by ulfh** [comment]
This appears to have been resolved as of version 1.1.1. Closing.
